src/tf_mcp_server/core/server.py

```diff
@@ -653,6 +655,109 @@ async def run_conftest_plan_validation(
                 }
             }
     
+    # ==========================================
+    # AZTFEXPORT TOOLS (DYNAMIC HELP APPROACH)
+    # ==========================================
+    
+    @mcp.tool("get_aztfexport_help")
+    async def get_aztfexport_help(
+        command: str = Field("", description="Optional command to get help for. Leave empty for main help showing all available commands and their descriptions.")
+    ) -> Dict[str, Any]:
+        """
+        Get dynamic help information directly from the aztfexport tool.
+        
+        This tool fetches live help information from aztfexport, ensuring documentation 
+        is always current and accurate. Use this before executing aztfexport commands.
+        
+        Recommended workflow:
+        1. Call get_aztfexport_help() to discover all available commands
+        2. Call get_aztfexport_help(command="<command_name>") for specific command help
+        3. Use run_aztfexport() with the appropriate arguments
+        
+        Args:
+            command: Optional command name to get detailed help for. The main help 
+                    (when command is empty) will show all available commands.
+                
+        Returns:
+            Live help text from aztfexport tool with all current options and usage.
+            The help content is fetched directly from aztfexport, ensuring it's always 
+            up-to-date with the installed version.
+        """
+        try:
+            result = aztfexport_runner.get_help(command)
+            return result
+        except Exception as e:
+            logger.error(f"Error getting aztfexport help: {e}")
+            return {
+                'success': False,
+                'error': f'Failed to get aztfexport help: {str(e)}',
+                'command': command
+            }
+        
+    @mcp.tool("run_aztfexport")
+    async def run_aztfexport(
+        command_args: str = Field(..., description="Complete aztfexport command arguments as you would type in terminal. CRITICAL: Positional arguments must come LAST after all flags. For QUERY mode, learn Azure Resource Graph syntax at portal.azure.com > Resource Graph Explorer. Use get_aztfexport_help() first to see available commands and options. Examples: 'rg --output-dir ./terraform myResourceGroup', 'res --non-interactive /subscriptions/.../resource', 'query --output-dir ./output \"type =~ \\'microsoft.automation/automationaccounts\\'\"'"),
```

Comment:
I noticed we're using a working directory instead of outputting the results as a string. For the docker container, do you need to mount a specific file system path inside the container? If so, please let me know your requirements and I can add them to the readme file. Alternatively, we could output the content as a string to simplify things.

Reply:
The output consists of rich content including terraform azurerm or azapi provider assembly, multiple .tf config files, tf state files, and so on. So making the output as a string would not work. I regard in majority cases the output should be located at users' working directories. 

Providing the output location that will be in the MCP server running env is sorts of backup if the output fails to be stored in users' directory due to unexpected causes. Given I regard it as a backup solution, I do not treat it as a prioritized consideration where to host that output location in the mcp server as long as it works.